### PR TITLE
Remove train predictions

### PR DESCRIPTION
**Pull request by jmorel**
Companion PR to https://github.com/SubstraFoundation/substra-tools/pull/24

In order for this PR to be merged, the one in substra-tools will have to be merged first a new release (0.5.0) will have to be done.

### CODE REVIEW

File: substra/runner.py [hunk 1]
```diff
@@ -112,7 +111,7 @@ def _docker_run(docker_client, name, command, volumes, remove=True):
 
 
 def compute_train(docker_client, train_data_path, algo_path, fake_data_samples, outmodel_path,
-                  train_pred_path, local_path, train_opener_file, rank, inmodels, outmodel_file,
+                  local_path, train_opener_file, rank, inmodels, outmodel_file,
                   metrics_path):
```
Comment (samlesu) [inline substra/runner.py]: `metrics_path` could be removed ?

File: docs/ml_tasks.md [hunk 2]
```diff
@@ -56,7 +47,7 @@ For prediction, Substra creates a new docker container containing the algorithm
 
 Substra then runs the algorithm **in testing mode** and predictions are saved to the **pred** volume.
 
-At this point, the score computation step starts and works exactly like the one in training. 
+At this point, the score computation step starts and works exactly like the one in training.
```
Comment (samlesu) [inline docs/ml_tasks.md]: This is not true anymore.

File: substra/runner.py (hunk 1 above)
Comment (jmorel) [reply substra/runner.py]: You're absolutely right, I removed it.

File: docs/ml_tasks.md (hunk 2 above)
Comment (jmorel) [reply docs/ml_tasks.md]: Good catch! I updated this line with the wording I removed from the training section above.

**Comment by jmorel** [comment]
Fixes #89

**Pull request merged**